### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by TomasPeterec**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,84 +1,13 @@
...
-    // const {error} = validateCours(req.body)
-    // if(error) return res.status(400).send(error.details[0].message)
-       
-    console.log(updatedVote)
-    
-    // res.send(updatedVote)
-    //Return the updated userVote
 
-})
+// Mount route files
```
Comment (peterhudec) [inline index.js:7]: I would remove this comment because it's obvious what the code is doing.

File: knex.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const knex = require("knex");
 const connectedKnex = knex({
     client: "sqlite3",
     connection: {
-        filename:"votes02.db"
+        filename:"./db/votes.db"
```
Comment (peterhudec) [inline knex.js:5]: I think this should come from an env var.

File: mail-triggered.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const { VERIFY_EMAIL } = require('./constants')
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+const API_ROOT = process.env.API_ROOT_URL
+
+const sendVerificationEmail = async (email, newToken) => {
+  try {
+    await sgMail.send({
+      to: email, // Change to your recipient
+      from: 'info@tomaspeterec.sk', // Change to your verified sender
```
Comment (peterhudec) [inline mail-triggered.js:12]: I would remove these comments:
```suggestion
      to: email,
      from: 'info@tomaspeterec.sk',
```

**Review by peterhudec — CHANGES_REQUESTED** [review]
It looks good, but I left a couple of suggestions.

File: index.js (hunk 1 above)
Comment (TomasPeterec) [reply index.js:7]: Later 💯

File: knex.js (hunk 2 above)
Comment (TomasPeterec) [reply knex.js:5]: Later 👍

**Review by peterhudec — APPROVED** [review]
Ok, now it looks great!

**Pull request merged**